src/main/resources/config/liquibase/changelog/20240626200000_changelog.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog https://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+    <changeSet id="20240626200000_1" author="bbesrour">
+        <!-- Create the new build_job_config table -->
+        <createTable tableName="programming_exercise_build_config">
+            <column name="id" type="BIGINT" autoIncrement="true">
+                <constraints primaryKey="true"/>
+            </column>
+            <column name="sequential_test_runs" type="BOOLEAN"/>
+            <column name="branch" type="VARCHAR(128)"/>
+            <column name="build_plan_configuration" type="longtext"/>
+            <column name="build_script" type="longtext"/>
+            <column defaultValueBoolean="false" name="checkout_solution_repository" type="BOOLEAN"/>
```

Comment:
Does `defaultValueBoolean` actually make the column non-nullable? Or is there a need to specify this constraint like below?

Reply:
I don't think it makes it non-nullable. But this was copied from `00000000000000_initial_schema.xml`